docs/advanced/miscellaneous.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Miscellaneous"
+summary: "Extra Stuff"
+draft: false
+toc: true
+sidebar_position: 5
+---
+
+## Using GDB with Linux AppImages (32 bit and 64 bit)
+
+This guide is useful if for example you'd like to debug a 32 bit crash but cannot build a 32 bit PCSX2.
+
+1. **Put your AppImage into a directory by itself**
+
+   We'll call the AppImage `PCSX2.AppImage` and then navigate your terminal to this directory..
+
+2. **Extract the contents of the AppImage**
+
+   You can do this by doing `./PCSX2.AppImage --appimage-extract`
+
+   A new directory called `squashfs-root` should have been created.
+
+3. **Run the AppImage normally**
+
+   `./PCSX2.AppImage`
+
+4. **Get the PID of the current PCSX2 instance**
+
+   You can do `pidof PCSX2`.
+
+   If there is more than one pid that shows up, you have more than one PCSX2 running!
+
+   We'll call your running pid `<PID>`
+
+5. **Attach GDB (during a fault or when PCSX2 is running normally, doesn't matter)**
+
+   Note: In my case I have to run gdb as `sudo`
+
+   `gdb squashfs-root/usr/bin/PCSX2 -p <PID>`
+
+   `set sysroot ./squashfs-root`
+
+   `set solib-search-path ./squashfs-root/usr/lib`
+
+   The last two commands are due to the fact that the AppImage runs in a sanboxed area.
+   It points gdb to where PCSX2 is going to look for libraries for example.
+
+The commands in-order are listed below:
+
+```sh
+./PCSX2.AppImage --appimage-extract
+
+./PCSX2.AppImage
+
+pidof PCSX2
+
+gdb squashfs-root/usr/bin/PCSX2 -p <PID>``
+
+set sysroot ./squashfs-root
+
+set solib-search-path ./squashfs-root/usr/lib
+```
+
+You now have GDB set up and connected to your PCSX2
+
+## Viewing Windows Minidumps on Other Platform
+
+If you're on Windows, the best way to view crash dumps is to open them in Visual Studio. Therefore, you can ignore these instructions if you are on Windows.
+
+:::warning
+You will need the users crash dump (.dmp) file, and the symbols (.pdb) and executable (.exe) of PCSX2 that is the _exact_ version the user was using when the crash dump was generated.
```

Comment:
"users" should be "user's", since it is for a single user but showing possession.